packages/form-js-editor/src/features/properties-panel/groups/CustomPropertiesGroup.js

```diff
@@ -52,7 +52,7 @@ export default function CustomPropertiesGroup(field, editField) {
       return editField(field, [ 'properties' ], removeKey(properties, key));
     };
 
-    const id = `${ field.id }-property-${ index }`;
+    const id = `property-${ index }`;
```

Comment:
Why do we remove this?

Reply:
@Skaiir ☝️

Reply:
@Skaiir this questions was still not answered

Reply:
Basically, all the other components were formatted this way, so this was an outlier. We don't usually supply the field id here.

Reply:
And if god forbid we implement multi-select and multi-edit in the future, what would we even call this :D